src/view/com/util/post-ctrls/RepostButton.tsx

```diff
@@ -157,7 +157,6 @@ let RepostButton = ({
             </View>
             <BottomSheetButton
               label={_(msg`Cancel quote post`)}
-              onAccessibilityEscape={close}
```

Comment:
Is this necessary? (I never quite understood how this prop works)

Reply:
It isn't supported on the RNGH pressable unfortunately (causes a crash). From what I read about it, I think the native dialog actually handles this for us (i think it lets the user use two finger presses to like dismiss or something...might look into the prop more later)